test/e2e/network_test.go

```diff
@@ -443,9 +443,13 @@ func (tc *testContext) getPodIP(selector metav1.LabelSelector) (string, error) {
 // getWindowsServerContainerImage gets the appropriate WindowsServer image based on VXLAN port
 func (tc *testContext) getWindowsServerContainerImage() string {
 	var windowsServerImage string
-	if tc.hasCustomVXLAN {
-		// If we're using a custom VXLANPort we will use 2022
+	if tc.CloudProvider.GetType() == config.VSpherePlatformType {
+		// If we're on vSphere we will use 2022
 		windowsServerImage = "mcr.microsoft.com/powershell:lts-nanoserver-ltsc2022"
+	} else if tc.CloudProvider.GetType() == config.NonePlatformType {
+		// TODO: On platform=none we have to use 2004 since the job points to the WS2004 golden image via release repo
+		// When the release repo is updated to use 2022, this clause should be squashed with above - `tc.hasCustomVXLAN`
+		windowsServerImage = "mcr.microsoft.com/powershell:lts-nanoserver-2004"
```

Comment:
doesn't seem necessary to do this over 2 different commits
you may have made this a separate one so that this commit can be reverted when the change is made, personally i think doing  	
```if tc.CloudProvider.GetType() == config.VSpherePlatformType && 
tc.CloudProvider.GetType() == config.VSpherePlatformType == config.NonePlatformType {
```
would be better as it makes the platforms used more explicit, instead of obfuscating things with the vxlan port check
so personally i think you should just squash this commit with the previous one

Reply:
@sebsoto I don't understand the suggested code. Shouldn't it be 
```
if tc.CloudProvider.GetType() == config.VSpherePlatformType || 
tc.CloudProvider.GetType() == config.NonePlatformType {
```